client/signup/controller.js

```diff
@@ -245,6 +246,17 @@ export default {
 				experiment.variationName === 'treatment_scrambled';
 		}
 
+		if ( isOnboardingFlow && wpcom.isTokenLoaded() ) {
```

Comment:
What is `wpcom.isTokenLoaded()`? Let's make sure the user is logged-in before providing an assignment too!

Reply:
Yes, that `wpcom.isTokenLoaded()` is true only if the user is logged in.
We noticed that `userLoggedIn` doesn't get set to true if the user sign up. It is true if the user is already logged in while going through the flow.